app/controllers/application_controller.rb

```diff
@@ -1,24 +1,2 @@
 class ApplicationController < ActionController::API
-  # For APIs, you may want to use :null_session instead.
```

Comment:
I would not delete this lines, mostyle because is has some kind of configuration, for example: `protect_from_forgery with: :null_session` that may caused troubles in the long run if the user is not aware of CSRF tokens

Reply:
Done :) .